corehq/apps/style/templates/style/partials/form_list_form.html

```diff
@@ -3,7 +3,7 @@
 {% load i18n %}
 
 <form id="table-form"
-      action="."
+      action="../bootstrap3/partials"
```

Comment:
i have a feeling that might have accidentally gotten in there crap due to a command+c mishap. thanks for the catch